opm/simulators/flow/FlowGenericProblem_impl.hpp

```diff
@@ -107,7 +106,6 @@ serializationTestObject(const EclipseState& eclState,
     result.solventRsw_ = {18.0};
     result.polymer_ = PolymerSolutionContainer<Scalar>::serializationTestObject();
     result.micp_ = MICPSolutionContainer<Scalar>::serializationTestObject();
-    result.mixControls_ = MixingRateControls<FluidSystem>::serializationTestObject(schedule);
```

Comment:
Do we need this also for FlowProblem?

Reply:
There is no test for a full blown FlowProblem, only for the GenericFlowProblem (see test_RestartSerialization.cpp). Hence, there is no place to stick this as such.

I'd suggest adding a test for the MixingRateControls and leave it at that. Adding a test for the full problem class is.. painful.

Reply:
Thanks for the information.  It is a good idea to suggest a test for MixingRateControls, but I am not familiar with implementation of MixingRateControls.

Reply:
see https://github.com/OPM/opm-simulators/pull/5543

Reply:
Thanks. I will wait for that PR goes in.